package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need this file?

Reply:
Yes.  If you use aasvg, you need this so that it builds for people who don't have it installed globally, at a minimum.